diff -urN screem-0.4.1/Makefile.in screem-0.4.1-patch/Makefile.in
--- screem-0.4.1/Makefile.in	Wed Mar 14 21:50:14 2001
+++ screem-0.4.1-patch/Makefile.in	Sun Jul  7 00:53:28 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @LIBGLADE_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
diff -urN screem-0.4.1/plugins/Makefile.in screem-0.4.1-patch/plugins/Makefile.in
--- screem-0.4.1/plugins/Makefile.in	Wed Mar 14 21:50:35 2001
+++ screem-0.4.1-patch/plugins/Makefile.in	Sun Jul  7 00:55:34 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @GDK_PIXBUF_CFLAGS@ @LIBGLADE_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
diff -urN screem-0.4.1/plugins/colourWizard/Makefile.in screem-0.4.1-patch/plugins/colourWizard/Makefile.in
--- screem-0.4.1/plugins/colourWizard/Makefile.in	Wed Mar 14 21:50:53 2001
+++ screem-0.4.1-patch/plugins/colourWizard/Makefile.in	Sun Jul  7 00:56:14 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @GDK_PIXBUF_CFLAGS@ @LIBGLADE_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
diff -urN screem-0.4.1/plugins/cssWizard/Makefile.in screem-0.4.1-patch/plugins/cssWizard/Makefile.in
--- screem-0.4.1/plugins/cssWizard/Makefile.in	Wed Mar 14 21:51:02 2001
+++ screem-0.4.1-patch/plugins/cssWizard/Makefile.in	Sun Jul  7 00:56:46 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @GDK_PIXBUF_CFLAGS@ @LIBGLADE_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
diff -urN screem-0.4.1/plugins/entityWizard/Makefile.in screem-0.4.1-patch/plugins/entityWizard/Makefile.in
--- screem-0.4.1/plugins/entityWizard/Makefile.in	Wed Mar 14 21:50:54 2001
+++ screem-0.4.1-patch/plugins/entityWizard/Makefile.in	Sun Jul  7 00:57:30 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @GDK_PIXBUF_CFLAGS@ @LIBGLADE_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
diff -urN screem-0.4.1/plugins/formWizard/Makefile.in screem-0.4.1-patch/plugins/formWizard/Makefile.in
--- screem-0.4.1/plugins/formWizard/Makefile.in	Wed Mar 14 21:51:03 2001
+++ screem-0.4.1-patch/plugins/formWizard/Makefile.in	Sun Jul  7 00:58:16 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @GDK_PIXBUF_CFLAGS@ @LIBGLADE_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
diff -urN screem-0.4.1/plugins/frameWizard/Makefile.in screem-0.4.1-patch/plugins/frameWizard/Makefile.in
--- screem-0.4.1/plugins/frameWizard/Makefile.in	Wed Mar 14 21:51:05 2001
+++ screem-0.4.1-patch/plugins/frameWizard/Makefile.in	Sun Jul  7 00:58:47 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @GDK_PIXBUF_CFLAGS@ @LIBGLADE_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
diff -urN screem-0.4.1/plugins/galleryWizard/Makefile.in screem-0.4.1-patch/plugins/galleryWizard/Makefile.in
--- screem-0.4.1/plugins/galleryWizard/Makefile.in	Wed Mar 14 21:50:55 2001
+++ screem-0.4.1-patch/plugins/galleryWizard/Makefile.in	Sun Jul  7 00:59:23 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @GDK_PIXBUF_CFLAGS@ @LIBGLADE_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
diff -urN screem-0.4.1/plugins/imageWizard/Makefile.in screem-0.4.1-patch/plugins/imageWizard/Makefile.in
--- screem-0.4.1/plugins/imageWizard/Makefile.in	Wed Mar 14 21:50:56 2001
+++ screem-0.4.1-patch/plugins/imageWizard/Makefile.in	Sun Jul  7 01:00:25 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @LIBGLADE_CFLAGS@ @GDK_PIXBUF_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
diff -urN screem-0.4.1/plugins/linkWizard/Makefile.in screem-0.4.1-patch/plugins/linkWizard/Makefile.in
--- screem-0.4.1/plugins/linkWizard/Makefile.in	Wed Mar 14 21:50:58 2001
+++ screem-0.4.1-patch/plugins/linkWizard/Makefile.in	Sun Jul  7 01:00:49 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @LIBGLADE_CFLAGS@ @GDK_PIXBUF_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
diff -urN screem-0.4.1/plugins/mailWizard/Makefile.in screem-0.4.1-patch/plugins/mailWizard/Makefile.in
--- screem-0.4.1/plugins/mailWizard/Makefile.in	Wed Mar 14 21:50:59 2001
+++ screem-0.4.1-patch/plugins/mailWizard/Makefile.in	Sun Jul  7 01:01:23 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @LIBGLADE_CFLAGS@ @GDK_PIXBUF_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
diff -urN screem-0.4.1/plugins/scripts/Makefile.in screem-0.4.1-patch/plugins/scripts/Makefile.in
--- screem-0.4.1/plugins/scripts/Makefile.in	Wed Mar 14 21:51:06 2001
+++ screem-0.4.1-patch/plugins/scripts/Makefile.in	Sun Jul  7 01:01:47 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @GDK_PIXBUF_CFLAGS@ @LIBGLADE_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
diff -urN screem-0.4.1/plugins/ssiWizard/Makefile.in screem-0.4.1-patch/plugins/ssiWizard/Makefile.in
--- screem-0.4.1/plugins/ssiWizard/Makefile.in	Wed Mar 14 21:51:01 2001
+++ screem-0.4.1-patch/plugins/ssiWizard/Makefile.in	Sun Jul  7 01:02:13 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @GDK_PIXBUF_CFLAGS@ @LIBGLADE_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
diff -urN screem-0.4.1/plugins/tableWizard/Makefile.in screem-0.4.1-patch/plugins/tableWizard/Makefile.in
--- screem-0.4.1/plugins/tableWizard/Makefile.in	Wed Mar 14 21:51:00 2001
+++ screem-0.4.1-patch/plugins/tableWizard/Makefile.in	Sun Jul  7 01:02:38 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @GDK_PIXBUF_CFLAGS@ @LIBGLADE_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
diff -urN screem-0.4.1/plugins/uploadWizard/Makefile.in screem-0.4.1-patch/plugins/uploadWizard/Makefile.in
--- screem-0.4.1/plugins/uploadWizard/Makefile.in	Wed Mar 14 21:50:51 2001
+++ screem-0.4.1-patch/plugins/uploadWizard/Makefile.in	Sun Jul  7 01:03:14 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @GDK_PIXBUF_CFLAGS@ @LIBGLADE_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
diff -urN screem-0.4.1/src/Makefile.in screem-0.4.1-patch/src/Makefile.in
--- screem-0.4.1/src/Makefile.in	Wed Mar 14 21:50:33 2001
+++ screem-0.4.1-patch/src/Makefile.in	Sun Jul  7 01:06:40 2002
@@ -65,7 +65,7 @@
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
 CC = @CC@
-CFLAGS = @CFLAGS@
+CFLAGS = @CFLAGS@ @GDK_PIXBUF_CFLAGS@ @LIBGLADE_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
 DATADIRNAME = @DATADIRNAME@
 DLLTOOL = @DLLTOOL@
