--- screem-0.12.1/screem.desktop.in	2004-09-17 21:56:15.000000000 +0000
+++ screem-0.12.1.new/screem.desktop.in	2004-11-23 14:41:57.082141128 +0000
@@ -3,8 +3,10 @@
 _Comment=Website editor and management system
 Exec=screem %U
 Icon=screem.png
-Terminal=0
+Terminal=false
 Type=Application
-Categories=Application;Development;
+Categories=GTK;GNOME;Application;Development;
 StartupNotify=true
 MimeType=text/html;application/xhtml+xml;text/xml;text/sgml;application/x-php;application/x-asp;text/vnd.wap.wml;image/svg;text/x-javascript;text/javascript;application/x-screem;application/x-screem-tag-tree;text/css;text/plain;text/x-perl;text/x-java;image/svg;text/x-php;text/mathml;text/x-dtd;text/x-sql;text/*
+Encoding=UTF-8
+# vi: encoding=utf-8
--- screem-0.12.1/po/pl.po	2004-11-23 03:47:19.000000000 +0000
+++ screem-0.12.1.new/po/pl.po	2004-11-23 14:42:39.653669272 +0000
@@ -17,12 +17,10 @@
 "Content-Transfer-Encoding: 8bit\n"
 
 #: screem.desktop.in.h:1
-#, fuzzy
 msgid "Screem HTML/XML Editor"
-msgstr "Screem: edytor szablonów"
+msgstr "Edytor HTML/XML Screem"
 
 #: screem.desktop.in.h:2
-#, fuzzy
 msgid "Website editor and management system"
 msgstr "Program do edycji i zarz±dzania stronami www"
 
